### PR TITLE
fix: ugly error when front matter has invalid yaml

### PR DESCRIPTION
**Pull request by quantum-ernest**
Resolves: #54

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func Load(rootDir string) (*Config, error) {
 
 	err = yaml.Unmarshal(yamlContent, &config.overrides)
 	if err != nil {
-		return nil, err
```
Comment (facundoolano) [inline config/config.go:84]: You are dropping the original yaml error here. Instead, wrap it so the details aren't lost: https://stackoverflow.com/a/61284074

Same for the other changes

**Review by facundoolano — CHANGES_REQUESTED** [review]

File: config/config.go (hunk 1 above)
Comment (quantum-ernest) [reply config/config.go:84]: check it out

Comment (facundoolano) [reply config/config.go:84]: as per the link above, I think you need to use `%w` rather than `%v`

Comment (quantum-ernest) [reply config/config.go:84]: ok

**Comment by facundoolano** [comment]
thanks

**Pull request merged**